projects/workshop/rick_and_morty_app/pubspec.yaml

```
@@ -0,0 +1,32 @@
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+  flutter_lints: ^5.0.0
+
+flutter:
+  uses-material-design: true
+
+  assets:
+    - assets/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)